printer_mods/ScramblerUSA/QuickReleaseClips/README.md

```diff
@@ -0,0 +1,75 @@
+# Quick Release Panel Clips
+
+Voron-style panel clips inspired by Annex-Engineering Denali printer's clips (https://github.com/Annex-Engineering/Denali/tree/master/STLs/Panel_Mounting/Side_Top_and_Rear_Panel_Quick_Release)
+
+![](./images/clips.jpg)
+
+I found Denali's clips too bulky and not really fitting to the overall style of Voron printer, so I designed these.
+
+## Parts Required
+
+None! You can even free up a few M8 screws and hammerhead nuts you are currently using.
```

Comment:
M3x8 screws, not M8 screws.

Reply:
D'oh! That's exactly what I meant to say.